package proto_ff;

import "yd_fieldoptions.proto";


message closetestActivity
{
	optional int32 ID = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 Type = 2[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 TypeParam = 3[(yd_fieldoptions.field_cname) = "类型参数"];
	optional int64 BoxId = 4[(yd_fieldoptions.field_cname) = "奖励id"];
}


message Sheet_closetestActivity
{
	repeated closetestActivity closetestActivity_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message closetestRebate
{
	optional int32 Day = 1[(yd_fieldoptions.field_cname) = "天数"];
	optional int32 ReturnParamA = 2[(yd_fieldoptions.field_cname) = "返还钻石系数"];
	optional int32 ReturnParamB = 3[(yd_fieldoptions.field_cname) = "返还绑钻系数"];
}


message Sheet_closetestRebate
{
	repeated closetestRebate closetestRebate_List = 1[(yd_fieldoptions.field_arysize)=20];
}
